app/Makefile

```diff
@@ -131,7 +131,20 @@ endif
 ############
 # Platform
 
-DEFINES       += PRINTF\(...\)=
+ifndef DEBUG
+	DEBUG = 0
+endif
```

Comment:
equivalent to 
DEBUG ?= 0

Reply:
@fbeutin-ledger I find it more explicit. If you really want I can change it to what you suggest.

Reply:
I prefer `?=` as it's more standard but as you want

Reply:
Ok I'll fix it.

Reply:
Done.